db/migrate/20170221195674_tidy_current_roles.rb

```diff
@@ -21,34 +21,10 @@ def up
   def process_existing(original_name, role, permission_names)
     diff = role.permission_diff permission_names
     return if diff.empty?
-    rename_existing role, original_name
+    SeedHelper.rename_existing_role role, original_name
     create_from_seeds original_name, permission_names
   end
 
-  def rename_existing(role, original_name)
```

Comment:
I wonder whether we shouldn't keep the method copy in here. The problem I see with this is that if later someone changes the implementation in seed helper, he/she would also change a the behavior of a migration that might or might not be applied. This could be easily missed but could result in users having different results.